webproject/src/main/webapp/js/room-my-order.js

```diff
@@ -119,23 +119,7 @@ async function addOrder() {
     window.location.reload();
 }
 
-async function deleteOrder(orderID) {
```

Comment:
is this supposed to be deleted?

Reply:
Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!

Reply:
Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!

Reply:
Thank you so much for spotting this, I hadn't realised! It was deleted because when I made this redirect PR, I mistakenly included a separate Delete Order PR inside this PR before the Delete Order PR was merged. I did not think that after the Delete Order PR was merged, it would delete as well. I will fix this!